lib/samples/style_graphics_with_symbols/style_graphics_with_symbols.dart

```diff
@@ -0,0 +1,336 @@
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:flutter/material.dart';
+import '../../utils/sample_state_support.dart';
+
+class StyleGraphicsWithSymbols extends StatefulWidget {
+  const StyleGraphicsWithSymbols({super.key});
+
+  @override
+  State<StyleGraphicsWithSymbols> createState() =>
+      _StyleGraphicsWithSymbolsState();
+}
+
+class _StyleGraphicsWithSymbolsState extends State<StyleGraphicsWithSymbols> with SampleStateSupport {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
+
+  final _graphicsOverlay = GraphicsOverlay();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Stack(
+        children: [
+          ArcGISMapView(
+            controllerProvider: () => _mapViewController,
+            onMapViewReady: onMapViewReady,
+          ),
+          // Display a progress indicator and prevent interaction until state is ready.
+          Visibility(
+            visible: !_ready,
+            child: const SizedBox.expand(
+              child: ColoredBox(
+                color: Colors.white30,
+                child: Center(child: CircularProgressIndicator()),
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  void onMapViewReady() async {
+    // Create the map.
+    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISOceans);
+
+    // Create a point using x and y coordinates.
+    final point = ArcGISPoint(
+      x: 56.075844,
+      y: -2.681572,
+      spatialReference: SpatialReference.wgs84,
+    );
+
+    // Set the initial viewpoint of the map to the point and provide a scale.
+    map.initialViewpoint = Viewpoint.fromCenter(point, scale: 2000);
+
+    // Set the map to the mapview controller.
+    _mapViewController.arcGISMap = map;
+
+    // Add graphics overlay to the mapview controller.
+    _mapViewController.graphicsOverlays.add(_graphicsOverlay);
+
+    // Call functions to create the graphics.
+    _createPoints();
+    _createPolygon();
+    _createPolyline();
+    _createText();
+
+    // Update the extent to encompass all of the symbols.
+    _setExtent();
+
+    // Set the ready state variable to true to enable the sample UI.
+    setState(() => _ready = true);
+  }
+
+  void _createPoints() {
+    // Create a red circle simple marker symbol.
+    SimpleMarkerSymbol redCircleSymbol = SimpleMarkerSymbol(
+        style: SimpleMarkerSymbolStyle.circle, color: Colors.red, size: 10,);
```

Comment:
There's a comma near the end of the line here. Normally that would get reformatted by `dart format`. Normally we let VSCode run the formatter when it saves a file. I'm not sure if you need to configure VSCode to run the formatter, or if maybe it just got missed here for some other reason. But you could run `dart format` from the command line also.

Reply:
I've formatted it with the `dart format` command. Most of the time I use Android Studio to code and VS Code to debug. I'll make sure to configure the formatter in VS Code, that'll help as well.